sirepo/auth_role_moderation.py

```diff
@@ -22,11 +22,81 @@
 
 _cfg = None
 
+
+@sirepo.api_perm.require_adm
+def api_admModerate():
+
+    def _send_moderation_status_email(info):
+        approved_denied_subject = f'{info.app_name} Access Request '
```

Comment:
I'd vote for a single character variable here as it is a temporary that is used in the next couple of lines.